fe/fe-core/src/main/java/com/starrocks/connector/RemoteScanRangeLocations.java

```diff
@@ -14,19 +14,31 @@
 
 package com.starrocks.connector;
 
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
 import com.google.common.collect.Lists;
 import com.starrocks.analysis.DescriptorTable;
+import com.starrocks.analysis.Expr;
 import com.starrocks.catalog.HiveMetaStoreTable;
 import com.starrocks.catalog.HiveTable;
 import com.starrocks.catalog.HudiTable;
 import com.starrocks.catalog.PartitionKey;
 import com.starrocks.catalog.Table;
 import com.starrocks.common.Config;
+import com.starrocks.datacache.DatacacheExprRewriter;
```

Comment:
You'd better use the `DataCache` prefix to keep same name format with lake module.

Reply:
changed in next pr

Reply:
changed in #32222